src/server/server.pro

```diff
@@ -3,14 +3,14 @@ QT += dbus
 
 TEMPLATE = app
 
-TARGET = timed-qt5
+TARGET = timed-qt$${QT_MAJOR_VERSION}
```

Comment:
Actual timed service (daemon) could be without version. This way also systemd service could be shared between different versions. For the shared library, I see value in separating different major versions.

Reply:
Further, this would make setcaps-timed-qt6.sh duplicate of setcaps-timed-qt5.sh meaning you could just rename setcaps-timed-qt5.sh to setcaps-timed.sh